examples/panda3d-viewer-play.py

```diff
@@ -26,7 +27,8 @@
 def play_sample_trajectory():
     update_rate = 60
     cycle_time = 3
-    traj = np.repeat(np.matrix(talos.q0).T, cycle_time * update_rate, axis=1)
+    traj = np.repeat(talos.q0.reshape((nq,1)), cycle_time * update_rate, axis=1)
+    print traj.shape
```

Comment:
```python
from __future__ import print_function

print(traj.shape)
```
=> compatible with both 2 and 3

Reply:
The printing stuff was for debugging, I removed.